CONTRIBUTION.md

```diff
@@ -0,0 +1,77 @@
+<h1 align="center">Contributing to The Project </h1>
+
+<p align="center">
+  Welcome to the NotesJar repository! We appreciate your interest in contributing.
+</p>
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+- [Code Style](#code-style)
+- [Reporting Bugs](#reporting-bugs)
+- [Feature Requests](#feature-requests)
+- [Contribution Process](#contribution-process)
+- [Pull Request Guidelines](#pull-request-guidelines)
+- [License](#license)
+
+## Getting Started
+
+1. Fork the repository.
+2. Clone the forked repository to your local machine:
+    ```bash
+    git clone https://github.com/your-username/[project-name].git 
+    ```
+3. Create a new branch for your feature or bug fix:
+    ```bash
+    git checkout -b feature-name
+    ```
+4. Make your changes and commit them with a clear and concise message:
+    ```bash
+    git add .
+    git commit -m "Your clear and concise message"
```

Comment:
Add this blog link on `How to write Better Git Commit Messages` by `FreeCodeCamp` along with the credits.

Link to the post: [https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/)